en/components/grids_templates/filtering.md

```diff
@@ -20,11 +20,12 @@ _keywords: angular filter, ignite ui for angular, infragistics
 ---
 }
 
-# @@igComponent Quick Filtering Overview
+# Angular @@igComponent Filtering
+IgniteUI for Angular @@igComponent component provides three different filtering types - Quick filtering, [Excel style filtering](excel_style_filtering.md) and [Advanced filtering](advanced_filtering.md) which enable you to display only the records which meet specified criteria. The Material UI grid component in Ignite UI provides angular filter capabilities and extensive filtering API through the Data Container to which the @@igComponent is bound.
 
-Angular grid filtering enables you to display only the records which meet specified criteria. The Material UI grid component in Ignite UI provides angular filter capabilities and extensive filtering API through the Data Container to which the @@igComponent is bound.
 
-### Demo
+
+## Angular Filter Example
 
```

Comment:
Can you add a brief description of the example - let's establish that as a pattern for all first demo examples for each grid topic

Reply:
Done :)